subsys/testsuite/ztest/src/ztest.c

```diff
@@ -1321,15 +1326,57 @@ static int cmd_shuffle(const struct shell *sh, size_t argc, char **argv)
 
 static int cmd_run_suite(const struct shell *sh, size_t argc, char **argv)
 {
+	struct getopt_state *state;
+	int opt;
+	static struct option long_options[] = {{"repeat_iter", required_argument, 0, 'r'},
+		{0, 0, 0, 0}};
+	int opt_index = 0;
+	int val;
+	int opt_num = 0;
+	void *param = NULL;
+	int repeat_iter = 1;
```

Comment:
You could use comma-separated declarations here, e.g.
```c
int val;
int opt_index = 0, opt_num = 0, repeat_iter = 1;
void *param = NULL;
```
Of course, what to group up and what to leave separate is a preference, so that's a judgement call.